### PR TITLE
Scala 2.12.4

### PR DESCRIPTION
**Pull request by kubukoz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/scala.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Scala < Formula
 
   resource "src" do
     url "https://github.com/scala/scala/archive/v2.12.4.tar.gz"
-    sha256 "6b1cbcf12582edcc807b49f5b69c5fdbda672fbc2ac457044e4e15cc81234ca3"
+    sha256 "9d1eaf570f95204a8894ab941070354b1672904a903ae3d1b45df201ddd1ed7d"
```
Comment (kubukoz) [inline Formula/scala.rb]: Strange, `curl https://github.com/scala/scala/archive/v2.12.4.tar.gz | shasum -a 256` gave me the result I had submitted in the first place

Comment (ilovezfs) [reply Formula/scala.rb]: you're welcome :)

**Comment by ilovezfs** [comment]
@kubukoz Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by kubukoz** [comment]
You're welcome ;)

**Comment by shongscience** [comment]
I can't see the 'history' log:
"Sorry, this commit history is taking too long to generate."

I need to see the commit history to downgrade my scala accordingly with my old spark

**Pull request merged**